client/src/components/employee/AddEmployeeDialogue.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+      <div class="overlay" v-if="value"></div>
+
+      <div class="modal" v-if="value">
+        <h3>Add Employee</h3>
+
+        <form @submit.prevent="addEmployee">
+          <md-field :class="idMessageClass">
+            <label for="employee_id">ID</label>
+            <md-input id="employee_id" required v-model="id" maxlength="6"></md-input>
+            <span class="md-error">{{this.idErrorMessage}}</span>
+          </md-field>
+
+          <md-field :class="nameMessageClass">
+            <label for="employee_name">Full ame</label>
+            <md-input id="employee_name" required v-model="name" maxlength="50"></md-input>
+            <span class="md-error">{{this.nameErrorMessage}}</span>
+          </md-field>
+
+          <md-button class="md-raised" type="cancel" @click="$emit('input', false)">CANCEL</md-button>
+          <md-button class="md-raised md-primary" :disabled="!formValidated" type="submit">SUBMIT</md-button>
```

Comment:
`!formValidated` poor naming... change to `!isFormValid`.

Reply:
changed